dali/base/dafdesc.cpp

```diff
@@ -1353,14 +1358,26 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
                 *_interleaved = interleaved;
         }
 
-        const char * historyStr = "History";
-        if (mb.remaining() > strlen(historyStr))
+        // If we have more serialised data in MemoryBuffer
+        if (mb.remaining() > sizeof(extendedVersion))
```

Comment:
minor: Should that be >= ?
In reality for this version it will always be >, but..